web/themes/joinup/joinup_theme.theme

```diff
@@ -89,11 +103,27 @@ function joinup_theme_preprocess_page(&$variables) {
 
     if (in_array($rdf_type, ['collection', 'solution'])) {
       $variables['has_vertical_tabs'] = TRUE;
+      $is_form_container = TRUE;
+    }
+  }
+
+  // Check pattern for add pages.
+  $form_pattern = '.add';
```

Comment:
Looks a bit unclear and unstable pattern. Can you maybe think of a regex or something that depends on specific elements rather than length of a string?